product_variant_attribute_name_manager/models/product_product.py

```diff
@@ -0,0 +1,13 @@
+from odoo import fields, models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    product_template_variant_value_ids = fields.Many2many(
+        domain=[
+            "|",
```

Comment:
Note that this can conflict between two modules not aware of each other.

Reply:
I don't understand exactly what you are referring to.